packages/_vue3-migration-test/src/components/test-base-suggestions.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <BaseSuggestions :suggestions="suggestions">
+    <template #default="{ suggestion }">
+      <span>{{ suggestion.query }}</span>
+    </template>
+  </BaseSuggestions>
+</template>
+
+<script setup>
+  import BaseSuggestions from '../../../x-components/src/components/suggestions/base-suggestions.vue';
+
+  const suggestions = [
```

Comment:
You can use the stub factory we have in `x-components` to avoid creating your own the data to render `const suggestions = getQuerySuggestionsStub('chip', 5);`